toolkit/phone-assessment-interviews/email-template-app-first-interview-sched.md

```diff
@@ -7,18 +7,29 @@ phase: phone-assessment-interviews
 title: Template - Email to Applicants Requesting First Interview
 ---
 
-Dear Applicant,
+Hello,
 
-Thank you for your interest in the `{POSITION-NAME}` position. You have moved forward from resume review. You're now invited to a phone interview with a subject matter expert to further assess your qualifications. This interview will last 30-60 minutes. If you move forward from this interview, you will have one additional subject matter expert qualifications interview. If you're referred for selection, you may be asked to participate in additional interviews with hiring managers.
+Thank you for your interest in the `{POSITION-NAME}` position. You have moved forward from the {PRIOR PHASE}. You’re now invited to a phone interview with a subject matter expert to further assess your qualifications. This interview will take about {TIME OF INTERVIEW} to complete. 
 
-Here are some key details about the position you've applied for:
+{IF THIS IS ONE OF TWO INTERVIEWS: If you move forward from this interview, you will have one additional subject matter expert qualifications interview.}
+{IF ONLY DOING ONE INTERVIEW: If you’re referred for selection, you may be asked to participate in additional interviews with hiring managers.}
```

Comment:
should say: If you’re referred for selection this means you have been deemed qualified. After that, you will likely be asked to participate in an additional interview with the hiring manager or multiple hiring managers if multiple offices are making selections from this job announcement.}

Reply:
maybe this should be at the end of the email? or else it looks like the sentence below regarding structured interviews applies to the hiring manager interview

Reply:
Will try to submit compromise text.